inc/namespace.php

```diff
@@ -61,6 +69,7 @@ function bootstrap() {
 }
 
 function on_http_request_args( $args, $url ) {
```

Comment:
Undocumented functions make @rmccue sad.

Reply:
That wasn't me!